sources/models/src/lib.rs

```diff
@@ -140,6 +140,14 @@ struct ContainerImage {
     superpowered: bool,
 }
 
+// Network settings. These settings will affect host service components' network behavior
+#[model]
+struct NetworkSettings {
+    http_proxy: Url,
+    https_proxy: Url,
+    no_proxy: Vec<Url>,
```

Comment:
You had mentioned making this support CIDR notation, but it doesn't seem like there's consistent support across the services we care about for CIDRs/wildcards/suffixes/etc.  Perhaps this should just be `Vec<SingleLineString>`?